File: C++/4. Estructuras Repetitivas/Ejercicio #1.cpp
/*1. Realiza un programa que solicite al ususario estandar un entero del 1 al 10 y muestre en
la salida estandar su tabla de multiplicar*/

#include<iostream>

using namespace std;

int main()
{
    int N = 0;

    cout<<"Ingrese u  valor del 1 al 10: ";cin>>N;

    for (int i = 1; i < 10; i++)
    {
        cout<<i<<" x "<<N<<" = "<<i*N<<endl;
    }
    
    return 0;
}

File: 3. Python/4. Bucles/Ejercicio #1.py
'''
 1. Realiza un programa que solicite al usuario estándar un entero del 1 al 10 y muestre en la salida estándar su tabla de multiplicar
'''

Entero = int(input(f"Ingrese un numero: "))

for i in range(1,10):
    print(f"{Entero} * {i} = {Entero*i}")
